erpnext/selling/doctype/quoting_sheet/quoting_sheet.json

```
@@ -0,0 +1,188 @@
+{
+ "autoname": "QUOTING-SHEET-.#####",
+ "creation": "2021-05-26 22:47:35.747870",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "item_code",
+  "item_name",
+  "customer",
+  "column_break_4",
+  "currency",
+  "conversion_rate",
+  "qty",
+  "section_break_8",
+  "rm_items",
+  "rm_cost",
+  "section_break_11",
+  "packaging_charges",
+  "column_break_13",
+  "shipping_cost",
+  "section_break_15",
+  "profit_markup",
+  "section_break_16",
+  "total_price",
+  "column_break_17",
+  "price_per_unit",
+  "amended_from",
+  "calculate"
+ ],
+ "fields": [
+  {
+   "fieldname": "item_code",
+   "fieldtype": "Link",
+   "in_list_view": 1,
+   "label": "Item",
+   "options": "Item",
+   "reqd": 1
+  },
+  {
+   "fetch_from": "item_code.item_name",
+   "fieldname": "item_name",
+   "fieldtype": "Data",
+   "label": "Item Name",
+   "read_only": 1
+  },
+  {
+   "fieldname": "amended_from",
+   "fieldtype": "Link",
+   "label": "Amended From",
+   "no_copy": 1,
+   "options": "Quoting Sheet",
+   "print_hide": 1,
+   "read_only": 1
+  },
+  {
+   "default": "1",
+   "fieldname": "qty",
+   "fieldtype": "Float",
+   "label": "Quantity",
+   "reqd": 1
+  },
+  {
+   "fieldname": "rm_items",
```

```suggestion
   "fieldname": "raw_material_items",
```
- Use precise variable names.